### PR TITLE
aci.py: Add use_proxy support

### PR DESCRIPTION
**Pull request by dagwieers**
##### SUMMARY
This PR includes:
- Support for the use_proxy parameter
- Additional notes related to proxy support and use_proxy parameter

##### ISSUE TYPE

 - Feature Pull Request

##### COMPONENT NAME
module_utils/aci.py

##### ANSIBLE VERSION
v2.4

### CODE REVIEW

**Comment by dagwieers** [comment]
cc: @schunduri

File: lib/ansible/utils/module_docs_fragments/aci.py [hunk 1]
```diff
@@ -42,6 +42,11 @@ class ModuleDocFragment(object):
     description:
     - The socket level timeout in seconds.
     default: 30
+  use_proxy:
+    description:
+      - If C(no), it will not use a proxy, even if one is defined in an environment variable on the target hosts.
+    default: 'yes'
+    type: bool
```
Comment (resmo) [inline lib/ansible/utils/module_docs_fragments/aci.py:49]: missing `version_added: '2.4'`

Comment (dagwieers) [reply lib/ansible/utils/module_docs_fragments/aci.py:49]: Well, there were no ACI modules in v2.4, so everything is new.

Comment (resmo) [reply lib/ansible/utils/module_docs_fragments/aci.py:49]: ah, my bad.

Comment (dagwieers) [reply lib/ansible/utils/module_docs_fragments/aci.py:49]: No worries !

**Pull request merged**